collector/processors/odigossqldboperationprocessor/processor.go

```diff
@@ -51,12 +59,29 @@ func (sp *DBOperationProcessor) processTraces(ctx context.Context, td ptrace.Tra
 				}
 
 				// Detect the `db.operation.name` from the query text
-				operationName := DetectSQLOperationName(dbQueryText.AsString())
+				operationName, tableName := DetectSQLOperationAndTableName(dbQueryText.AsString())
 
+				// Used to build "{operation} {table}" span name
+				spanName := ""
 				// Only set the `db.operation.name` if the detected operation name is not "UNKNOWN"
 				if operationName != OperationUnknown {
 					span.Attributes().PutStr(string(semconv.DBOperationNameKey), operationName)
+					spanName = operationName
+				}
+
+				if tableName != OperationUnknown {
+					span.Attributes().PutStr(string(semconv.DBCollectionNameKey), tableName)
```

Comment:
Should we check if the attribute not exists before putting it?
We have a similar check for the opetation